pkgs/development/ocaml-modules/github/unix.nix

```
@@ -6,7 +6,9 @@ buildDunePackage {
   pname = "github-unix";
   inherit (github) version src;
 
-  duneVersion = "3";
+  postPatch = ''
+    substituteInPlace unix/dune --replace 'github bytes' github
```

This looks slightly weird and is inconsistent with other substitutions.
```suggestion
    substituteInPlace unix/dune --replace 'github bytes' 'github'
```